src/components/register/RegistrationForm.tsx

```diff
@@ -0,0 +1,125 @@
+'use client';
+import React from 'react';
+
+import * as Form from '@radix-ui/react-form';
+
+import InputFile from '@/components/ui/InputFile';
+
+interface RegistrationFormProps {
+  eventName: string;
+  requiresPayment: boolean;
+}
+
+const RegistrationForm: React.FC<RegistrationFormProps> = ({
+  eventName,
+  requiresPayment,
+}) => {
+  return (
+    <div className="flex flex-col justify-center">
+      <div className=" flex flex-col items-center border-y-2 py-5">
+        <h1 className="font-semibold text-2xl">{eventName}</h1>
+        <h1 className="font-semibold text-5xl">Registration Form</h1>
+      </div>
+      <div className="flex flex-col items-center mt-20">
+        <Form.Root
+          className="FormRoot"
+          action="/student/submit-registration"
```

Comment:
na test mo to sya na mag POST sya request sa backend?

Reply:
AY wala pa mag update ako after testing

Reply:
okee okeeess